tuner/tuner/candidate_gen_test.py

```diff
@@ -36,6 +38,180 @@ def remove_comments(mlir: str) -> str:
     )
 
 
+def test_get_td_spec_contraction(tuner_ctx: common.TunerContext) -> None:
```

Comment:
I wonder if we should use LIT for this... I think it's fine for now, but maybe we should revisit in the future